flyio/rush/rush-config.yml

```diff
@@ -0,0 +1,104 @@
+# NOTE: THIS FILE IS NOT USED -  all configuration options for reference only
+core:
+  enabled: true # enable httpd server
+  address: "" # ip address to bind (default: any)
+  shutdown_timeout: 30 # default is 30 second
+  port: "8088" # ignore this port number if auto_tls is enabled (listen 443).
+  worker_num: 0 # default worker number is runtime.NumCPU()
```

Comment:
Did we have this set to 0 on the other notification

Reply:
demo config